### DISCUSSION TITLE
Add ability to subscribe to activitypub feeds

### DISCUSSION THREAD

**Discussion by DraconicNEO** [spacecowboy/Feeder#266] [Ideas]
Maybe this is out of scope of the project but I feel like it would be cool to be able to subscribe to feeds from activitypub platforms in addition to RSS feeds. I think it would be a good idea since there may be situations where an activitypub platform has no RSS feed for whatever reason and being able to subscribe to the ActivityPub feed would allow users to view that in this reader. I had the idea because some people on Mastodon were talking about the idea of an RSS reader with Activitypub support and they mentioned one Reader which has considered it on the roadmap but never implemented.

**Comment by spacecowboy** [comment]
I'm not gonna say it's impossible, but seems unlikely.  

seems more relevant to use an activitypub client with a rss to activitypub bridge. 

or some server side software which can convert activitypub to rss
